release-pipeline.yml

```diff
@@ -110,6 +110,10 @@ stages:
               GOARCH=amd64 GOOS=linux go build -tags "netgo,se_integration" -o "$(root_dir)/azcopy_linux_se_amd64"
             displayName: 'Generate Linux AMD64 SE Integration'
 
+          - script: |
```

Comment:
GRPC feature can be triggered by CLI, do we need to build a seperate binary for that.

Reply:
It's more intended to be a separate build so a customer has to go out of their way, fully understanding we won't support it, in order to use it. The other point is so that we don't introduce new dependencies in the normal build. If they choose to use it, they should expect things to break and change.

By not including it in the normal build, we release nothing anybody could accidentally come to depend on without intentionally expecting a footgun.